### DISCUSSION TITLE
Clarify Licensing

### DISCUSSION THREAD

**Discussion by gregorybleiker** [Budibase/budibase#7541] [Help] [answered]
From https://budibase.com/pricing/ I can see that the "free cloud tier" has "1000 internal records per app". I don't understand what this means. I uploaded 3000 rows of test data without any problems, so it does not seem to refer to data rows in the internal db (and a 1000 would be very restrictive for that anyway). Can someone elaborate on this? Thanks!

**Comment by andz-bb** [comment, marked as answer]
Hey @gregorybleiker 

On the free cloud tier you can have up to four apps and the total limit for records across all of those apps is 4000, so 1000 per app if split across them evenly. There's no restriction on how these are distributed across the apps so if you only have one app you could upload up to 4000 records there. Our paid cloud hosted tiers are launching very soon and you should be able to see your account's usage and limits in more detail with that update. Hope this clears things up for you 😊
